Diplomatic.Android/Diplomatic.Android.csproj

```diff
@@ -86,18 +86,24 @@
     <AndroidResource Include="Resources\drawable\gavinbelson.png" />
     <AndroidResource Include="Resources\drawable\waltdisney.png" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Resources\drawable-hdpi\" />
-    <Folder Include="Resources\drawable-xhdpi\" />
-    <Folder Include="Resources\drawable-xxhdpi\" />
-    <Folder Include="Resources\drawable-xxxhdpi\" />
-    <Folder Include="Resources\drawable\" />
-  </ItemGroup>
+  <ItemGroup />
```

Comment:
I had one of these empty ItemGroups show up in one of my commits too: I have no idea what causes it, but they can safely be removed.

Reply:
Do eet <3